Code/JavaWrappers/gmwrapper/src-test/org/RDKit/SuppliersTests.java

```diff
@@ -167,6 +167,28 @@ public void test10Leak() {
                 if((i%1000)==0) System.err.printf("Done: %s\n",i);
             }
 	}
+  
+        @Test
+        public void test11GZstream() {
+          // NCI_aids_few.sdf.gz
+          File base = getRdBase();
+          File gzpath  = new File(base, "Code" + File.separator + "GraphMol" + File.separator +
+                                  "FileParsers" + File.separator + "test_data");
+          File fileN = new File(baseTestPath, "NCI_aids_few.sdf.gz");
```

Comment:
looks like you forgot to add this file

Reply:
I'm using the file in the FileParsers suite:

https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/FileParsers/test_data/NCI_aids_few.sdf.gz

Reply:
ah, sorry. I looked in the wrong place. Reviewer-error.